lib/page.js

```diff
@@ -181,6 +181,7 @@ var Page = function( page_path, options ){
           if (err) {
             var status = err.status || 500;
             var error  = 'Resource error: ' + err.message;
+            var isOptional = context.isPreview || resource.options.optional;
```

Comment:
We usually use underscore casing for variables, and camel casing for methods, in Solidus... (`is_optional`).